buf.yaml

```diff
@@ -398,3 +398,4 @@ breaking:
     # it's the best/least bad option, you can add it here and remove it in a
     # later PR:
     # - external/oops/oops.proto
+    - external/infra_proxy/infra_proxy.proto
```

Comment:
Consider adding a "TODO" comment here saying this needs to be removed in a follow-up PR.